jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -105,9 +106,21 @@ public static String printSmartLocators(IBaseElement el) {
             }
         });
     };
+    private static void fillAction(JAction1<String> action, String name) {
+        String prop = null;
+        try {
+            prop = getProperty(name);
+        } catch (Exception ignore) {}
+        if (prop == null) {
+            prop = "null";
+        }
+        logger.debug("fillAction(%s=%s)", name, prop);
+        PropertyReader.fillAction(action, name);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'PropertyReader.fillAction' due to existing static import 'com.epam.jdi.tools.PropertyReader.fillAction'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5570565)